packages/web/components/transactions/transaction-details/transaction-swap-details.tsx

```diff
@@ -11,18 +10,22 @@ import { IconButton } from "~/components/buttons/icon-button";
 import { Button } from "~/components/ui/button";
 import { EventName } from "~/config";
 import { useAmplitudeAnalytics, useTranslation } from "~/hooks";
+import { useTransactionHistory } from "~/hooks/use-transaction-history";
 import { theme } from "~/tailwind.config";
 import { formatPretty } from "~/utils/formatter";
 import { formatFiatPrice } from "~/utils/formatter";
 
-export const TransactionDetailsContent = ({
+export const TransactionSwapDetails = ({
   onRequestClose,
   isModal,
   transaction,
 }: {
   onRequestClose: () => void;
   isModal: boolean;
-  transaction: FormattedTransaction;
+  transaction: Extract<
```

Comment:
Are we sure we can't define this type anywhere, perhaps near useTransactionHistory? It's not super readable here